scripts/push-to-assets-branch

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+rm -rf /tmp/tmp-assets
+mkdir -p /tmp/tmp-assets
+cp -r assets/ index.yaml index.html sha256sum/ /tmp/tmp-assets
+if [[ -d charts ]]; then
+  cp -r charts/ /tmp/tmp-assets
+fi
+git checkout .
+git clean -fd
+git fetch ${1} ${2}
+git checkout ${2}
+cp -r /tmp/tmp-assets/* ./
+git add .
+git -c user.name="actions" -c user.email="actions@github.com" commit -m "Generated changes" || true
+git push ${1} ${2}
```

Comment:
I'm not a huge fan of including a `git push` directly as part of the scripts; I think this script should be changed from `push-to-assets-branch` to `generate-commit-in-assets-branch` with everything included in the current version of this script except for this line.

This would prevent a dev who accidentally ran this script from accidentally potentially pushing to the wrong branch.

This specific line could then be moved to the `run` field of the `.github/workflows/push.yml` e.g.
```
./scripts/generate-commit-in-assets-branch origin dev-v2.5-assets && git push origin dev-v2.5-assets
```

Reply:
:+1: